### PR TITLE
Removed link from image

### PR DESCRIPTION
**Pull request by Artemis6161**
# Fixes
Fixes https://github.com/creativecommons/cc-resource-archive/issues/197 by @Xaid-vfx

# Description
Resource images will no longer act as links. 

# Technical details
Removed the anchor tag around the resource image.

# Screenshots

Screenshot 2024-03-09 225858

https://github.com/creativecommons/cc-resource-archive/assets/78592463/24e62fb5-e033-4194-8128-d7d3bf370c95

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: docs/style.css
Comment (Xaid-vfx) [inline docs/style.css:1]: Make sure to not make any unnecessary changes in any file such as line additions

**Review by Xaid-vfx — CHANGES_REQUESTED** [review]

File: docs/Gemfile.lock
[the remark this answers is not in the document]
Comment (Artemis6161) [reply docs/Gemfile.lock:1]: The bundler was not installed properly.
now i have installed again but it is showing many changes in Gemfile.

**Comment by possumbilities** [comment]
@Artemis6161 You still need to remove changes to the `Gemfile.lock`

**Comment by Artemis6161** [comment]
Sure, I will work on it.

**Comment by Artemis6161** [comment]
I have made the changes, please review this PR. Thankyou

**Review by possumbilities — APPROVED** [review]
Wonderful thank you! ❤️

**Comment by Artemis6161** [comment]
It my pleasure, thank you so much.

**Pull request merged**